### DISCUSSION TITLE
12/13 안승민 블로그 2회차

### DISCUSSION THREAD

**Discussion by smahn9123** [Orm-camp/blog-feedback#51] [동글동글 4기 피드백]
https://comgu.tistory.com/entry/Python-List-곱하기-연산의-메모리-구조-분석1차원-list-다중-list
https://comgu.tistory.com/entry/Python-객체의-shallow-copy와-deep-copy
https://comgu.tistory.com/entry/Python-클로저Closure-함수
https://comgu.tistory.com/entry/Python-데코레이터Decorator-1-데코레이터-중첩-데코레이터
https://comgu.tistory.com/entry/Python-데코레이터Decorator-2-동적-데코레이터
https://comgu.tistory.com/entry/Python-데코레이터Decorator-3-클래스형-데코레이터

**Comment by Jungminchae** [comment]
- 시리즈로 작성해주셔서 연달아 읽기가 너무 좋네요 
- 적절한 예시를 잘 만드신 것 같습니다. 다른 블로그들 보다보면 예시가 너무 길고 복잡하면 잘 안보게 되더라구요.
- 그리고 SEO 관점에서 소제목 h2로 하고 하위 소제목을 h3로 하시면 더 좋은 걸로 알고 있어요.
![image](https://github.com/user-attachments/assets/8f823b6d-223f-49f4-b00d-eb66dc068790)
여기서 동적 데코레이터 -> h2
조건에 따라 데코레이터 함수의 동작을 동적으로 변경할 수 있는 예시 -> h3 
(예전에 저도 그렇게 배움)
- 꾸준히 작성하시는 모습 넘 좋습니다! 계속 화이팅

**Reply by smahn9123** [reply]
@Jungminchae 소제목 포맷 정하는 법에 대한 기준이 없었는데 알려주셔서 고맙습니다 :)
이미 작성한 글들부터 하나씩 수정해보겠습니다.

**Comment by devSeyong** [comment]
어제부터 데코레이터때문에 많이 힘들었는데 글 보면서 잘 공부해보도록 하겠습니다! 유익한 글 감사합니다!

**Reply by smahn9123** [reply]
감사합니다!
